File: counter.js
const numbers = Array.from({ length: 100 }, (_, index) => index + 1);

numbers.forEach((number) => {
  if (number % 3 === 0 && number % 5 === 0) {
    console.log("Hawaii");
  } else if (number % 3 === 0) {
    console.log("Pizza");
  } else if(number % 5 === 0) {
    console.log("Pineapple");
  } else {
    console.log(number);
  }
})

File: counter.rb
numbers = (1..100).to_a

numbers.each do |num|
  if num % 5 == 0 && num % 3 == 0
    puts "Pizza"
  elsif num % 5 == 0
    puts "Pineapple"
  elsif num % 3 == 0
    puts "Hawaii"
  else
    puts num
  end
end
